src/components/faktum/faktum-number/FaktumNumberReadOnly.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { BodyShort, Label } from "@navikt/ds-react";
+import { IQuizNumberFaktum } from "../../../types/quiz.types";
+import { IFaktumReadOnly } from "../Faktum";
+import { useSanity } from "../../../context/sanity-context";
+import { PortableText } from "@portabletext/react";
+import { HelpText } from "../../HelpText";
+import styles from "../Faktum.module.css";
+
+export function FaktumNumberReadOnly(props: IFaktumReadOnly<IQuizNumberFaktum>) {
+  const { faktum, showAllFaktumTexts } = props;
+  const { getFaktumTextById } = useSanity();
+  const faktumTexts = getFaktumTextById(faktum.beskrivendeId);
+
+  return (
+    <>
+      <Label>{faktumTexts ? faktumTexts.text : faktum.beskrivendeId}</Label>
```

Comment:
`<Label as="p">` 😊  (skal slutte å kommentere på denne, den gjelder jo for alle ReadOnly-komponentene)